src/unity/lib/extensions/model_base.hpp

```diff
@@ -95,14 +95,30 @@ class EXPORT model_base: public cppipc::ipc_object_base {
    * Loads a toolkit class previously saved at a particular version number.
    * Should raise an exception on failure.
    */
-  virtual void load_version(iarchive& iarc, size_t version);
+  virtual void load_version(iarchive& iarc, size_t version) {} ;
```

Comment:
Same comment as above.

Reply:
A comment note added to the empty implementations as per offline discussion.